src/System.Globalization/ref/System.Globalization.csproj

```diff
@@ -14,5 +14,9 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj">
```

Comment:
Lets have a work item tracking the removal of these ProjectReferences once we get packages with the changes.